### DISCUSSION TITLE
Seeed Studio XIAO RP2040

### DISCUSSION THREAD

**Discussion by Bosswhite12** [shanteacontrols/OpenDeck#318] [General]
Hello I have just tried one of these boards. As it has 4 analogue inputs I was hoping it might show 4 in the configurator but it only shows 3 like the official Pi Pico. Is there a way to use all 4 analogue inputs?
Thanks Nick

**Comment by paradajz** [comment]
RP2040 has only 3 analog inputs which you can use freely. 4th one is temperature sensor.

**Reply by Bosswhite12** [reply]
Hello Thank you for your quick reply. Having "Googled" a bit more as you said on the official Pi Pico the 4th input is used for a temperature sensor. But it looks like some alternative boards that use the RP2040 chip offer 4 analogue inputs like the Adafruit Feather RP2040 and the xiao 2040. Thanks Nick
